### PR TITLE
added strokeWidth & radius as params for progress chart

### PR DESCRIPTION
**Pull request by mateusarbex**
I felt the need to add these 2 params as I was tinkering with the progress chart and it didn't fit into the design of the project I was working on. These two params fixed most of the issues I've had

### CODE REVIEW

**Comment by Hermanya** [comment]
could you update the readme too?

File: src/progress-chart.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const Pie = require("paths-js/pie");
 
 class ProgressChart extends AbstractChart {
   render() {
-    const { width, height, style = {}, data, hideLegend } = this.props;
+    const { width, height, style = {}, data, hideLegend, strokeWidth = 16, radius = 32 } = this.props;
```
Comment (Hermanya) [inline src/progress-chart.js:10]: could you use `defaultProps`?

**Comment by mateusarbex** [comment]
Done!

**Comment by Hermanya** [comment]
Thank you for you contribution :+1: this is now available in v5.4.0

**Pull request merged**